packages/core/contracts/UnirepSocial.sol

```diff
@@ -28,10 +28,17 @@ contract UnirepSocial {
     // The amount of karma airdropped to user when user signs up and executes user state transition
     uint256 immutable public airdroppedReputation;
 
+    // The set of user names
+    uint256[] public userNames;
+
+
     // A mapping between user’s epoch key and if they request airdrop in the current epoch;
     // One epoch key is allowed to get airdrop once an epoch
     mapping(uint256 => bool) public isEpochKeyGotAirdrop;
 
+    // A mapping between username and if they're already claimed;
+    mapping(uint256 => bool) public isClaimed;
```

Comment:
Because this is the only state variable relating to usernames this should be named something related to usernames like `claimedUsernames` or even just `usernames`.

Reply:
Ah now I understand how set of usernames are represented by this mapping.